docs/src/man/getting_started.md

```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
+
+example :
```

Comment:
Missing initial upper case. Also, better not create a new paragraph just for this word.